pysrc/scripts/evaluation.py

```diff
@@ -35,7 +35,7 @@ def create_evaluation_csv(runs: MetanomeRunBatch, output_file: str, output_folde
 
     with open(output_csv, 'w') as csv_output:
         writer = csv.writer(csv_output, quoting=csv.QUOTE_ALL)
-        writer.writerow(['sampled_files', 'sampling_method', "sampling_rate", 'tp', 'fp', 'fn', 'precision', 'recall', 'f1'])
+        writer.writerow(['sampling_method', "sampling_rate", 'tp', 'fp', 'fn', 'precision', 'recall', 'f1'])
```

Comment:
What about the changes in this file?

Reply:
I felt that we never really care about the specific file names and as such there is no need to keep them in the output csv